### ISSUE TITLE
Does admin token need access to /search?

### ISSUE THREAD

**Issue by aljachimiak** [oddnetworks/oddworks#222]
While reading through the routes, I noticed [this restriction on the search endpoint](https://github.com/oddnetworks/oddworks/blob/master/lib/services/catalog/index.js#L151-L157).

````javascript
        router.all(
			'/search',
			middleware['request-authorize']({bus, audience: {
				get: ['platform']
			}}),
			CatalogSearchController.create({bus})
		);
````

Will an admin token need access to the search endpoint?

**Comment by blainsmith** [comment]
It shouldn't, but in reality admin should be allowed to do everything at any time.

**Comment by aljachimiak** [comment]
I"m going to interpret 👍  as 
"This endpoint needs to support an admin token".

I'll cue this up behind the DELETE functionality for admin tokens.

Please feel free to correct if I have misinterpreted.

-Al

**Comment by kixxauth** [comment]
yeah, there are use cases for admin using the /search endpoint. For example, to allow dashboard users to find their content.
